tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/QueueControlTest.java

```diff
@@ -1329,6 +1329,64 @@ public void testMoveMessage() throws Exception {
       session.deleteQueue(otherQueue);
    }
 
+   /**
+    *    Moving message from another address to a single "child" queue of a multicast address
+    *
+    *    <address name="ErrorQueue">
+    *             <anycast>
+    *                <queue name="ErrorQueue" />
+    *             </anycast>
+    *          </address>
+    *          <address name="parent.addr.1">
+    *             <multicast>
+    *                <queue name="child.queue.1" />
+    *                <queue name="child.queue.2" />
+    *                <queue name="child.queue.3" />
```

Comment:
Base on the test, this queue is never created/used. Should be removed.

Reply:
The rest looks good to me.

Reply:
Thank you for review, I removed the unused queue from comment.